package-lock.json

```diff
@@ -4637,8 +4637,7 @@
             "wordwrap": {
               "version": "0.0.3",
               "resolved": "https://registry.npmjs.org/wordwrap/-/wordwrap-0.0.3.tgz",
-              "integrity": "sha1-o9XabNXAvAAI03I0u68b7WMFkQc=",
-              "dev": true
```

Comment:
Huh? Looks like npm correctly handles the case when a package becomes used in `dependencies` and not only in `devDependencies`, but does not handle the case when the package is not required for `dependencies` anymore. Can you please revert this change manually?